src/games/arena/actions/index.ts

```diff
@@ -114,18 +114,20 @@ export const handleViewSubmissionAction = async (payload: SlackBlockKitPayload)
   }
 
   switch (callback_id) {
-    case ARENA_SECONDARY_ACTIONS.CREATE_OR_UPDATE_ZONE_DATA:
-      zoneRepository
-        .createOrUpdateZoneForm(userRequesting, values as ZoneData)
-        .then(async (response) => {
-          if (response?.type === 'error') {
-            await arenaNotifyEphemeral(
-              response.text ?? 'Something went wrong',
-              userRequesting.slackId!,
-              userRequesting.slackId!
-            );
-          }
-        });
+    case ARENA_SECONDARY_ACTIONS.CREATE_OR_UPDATE_ZONE_DATA: {
+      const response = await zoneRepository.createOrUpdateZoneForm(
+        userRequesting,
+        values as ZoneData
+      );
+
+      if (response?.type === 'error') {
+        await arenaNotifyEphemeral(
+          response.text ?? 'Something went wrong',
+          userRequesting.slackId!,
+          userRequesting.slackId!
+        );
+      }
+    }
```

Comment:
eslint error asking to add await

Reply:
Not 100% sure about this, but it looks it would work the same way. We need to double-check that from the slack commands.